frontend/src/metabase/home/components/Activity.jsx

```diff
@@ -528,8 +528,13 @@ export default class Activity extends Component {
           <div className="full flex flex-column">
             <div className="">
               {activity.length === 0 ? (
-                <div className="flex flex-column layout-centered mt4">
-                  <span className="QuestionCircle">!</span>
+                <div className="flex flex-column layout-centered my4">
+                  <span
+                    className="QuestionCircle"
+                    style={{ lineHeight: "normal" }}
```

Comment:
Set `line-height` to normal to counter the `line-height` set by the `<Card />` component.

Reply:
My inclination is we put `line-height: normal` in the `.QuestionCircle` rule found in `metabase/css/home.css`. The only other place we use the `QuestionCircle` class is in the dashboard empty state and it seems like it could theoretically be affected by a similar issue where some parent component (arguably erroneously) sets a `line-height` that its children inherit. And that way you can avoid the in-lined styling.

Reply:
Makes sense.